unstructured/partition/xlsx.py

```diff
@@ -11,17 +13,12 @@
 )
 from unstructured.file_utils.filetype import FileType, add_metadata_with_filetype
 from unstructured.partition.common import (
+    contains_emoji,
     exactly_one,
     get_last_modified_date,
     get_last_modified_date_from_file,
     spooled_to_bytes_io_if_needed,
 )
-from unstructured.utils import dependency_exists
-
-if dependency_exists("bs4"):
-    from lxml.html.soupparser import fromstring as html_string_parser
```

Comment:
actually, i think the intent of this if statement was to allow "minimal" installs that don'thave bs4 installed to use the "lesser" html_string_parser.

to honor that convention, i think the better thing to do would be to strip emoji chars, then use document_fromstring if bs4 is not installed. wdyt?

Reply:
Yes, it's reasonable to have `bs4` in a minimal unstructured install. That was already done in the previous PR for this issue - https://github.com/Unstructured-IO/unstructured/pull/1077/files.